### PR TITLE
TLS client certificate support

### PR DESCRIPTION
**Pull request by fgouteroux**
Hi, 

I want to add the tls client support needed when the verify_peer option is set tot true on rabbitmq.

Does it seems good ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,19 +129,27 @@ Usage:
   rabtap -h|--help
   rabtap info [--api=APIURI] [--consumers] [--stats] [--filter=EXPR] [--omit-empty] 
               [--show-default] [--mode=MODE] [--format=FORMAT] [-knv]
+              [--cert-file=CERTFILE] [--key-file=KEYFILE] [--ca-file=CAFILE]
   rabtap tap EXCHANGES [--uri=URI] [--saveto=DIR] [--format=FORMAT] [-jknsv]
+                       [--cert-file=CERTFILE] [--key-file=KEYFILE] [--ca-file=CAFILE]
```
Comment (jandelgado) [inline README.md:134]: we should name the options `--tls-key-file`, `--tls-cert-file` and `--tls-ca-file` to make it clear that these options are TLS related?

File: README.md [hunk 2]
```diff
@@ -213,6 +225,9 @@ Examples:
   rabtap info
   rabtap info --filter "binding.Source == 'amq.topic'" --omit-empty
   rabtap conn close "172.17.0.1:40874 -> 172.17.0.2:5672"
+
+  # use RABTAP_CERTFILE | RABTAP_KEYFILE | RABTAP_CAFILE environments variables
```
Comment (jandelgado) [inline README.md:229]: also we should name the environment variables like `RABTAP_TLS_xxx` for consistency.

File: cmd/rabtap/command_line.go [hunk 3]
```diff
@@ -220,7 +237,28 @@ func parseAPIURI(args map[string]interface{}) (string, error) {
 }
 
 func parseCommonArgs(args map[string]interface{}) commonArgs {
+	var certFile string
```
Comment (jandelgado) [inline cmd/rabtap/command_line.go:240]: please provide unit tests in command_line_test.go

**Review by jandelgado — COMMENTED** [review]
Good idea and valuable addition. Thank you. I've left somme comments. Please also add the unit tests.

In the meantime I need to check why the CI did not run on the MR....

**Comment by fgouteroux** [comment]
@jandelgado I just following your review, is it correct ?

**Comment by jandelgado** [comment]
The changes basically look good, but know as I got travis ci run again, there are now some build problems coming up. Please check the CI logs at travis and/or run the build locally using `make test` and `make run-broker` before in a second console.

Thanks for your contribution!

**Comment by fgouteroux** [comment]
@jandelgado it decrease the coverage as the getTLSConfig function is not tested. I 've no idea about implementing this check in unit tests.

**Pull request merged**